src/test/resources/uk/gov/gdx/datashare/controller/approvals/EventsControllerTest.getEvents gets events of expected shape.approved.txt

```diff
@@ -1 +1 @@
-PagedModel { content: [EntityModel { content: EventNotification(eventId=a5383689-1192-4078-a4a6-a611b0a34c6e, eventType=DEATH_NOTIFICATION, sourceId=a5383689-1192-4078-a4a6-a611b0a34c6e, dataIncluded=null, enrichmentFields=null, eventData=DeathNotificationDetails(registrationDate=null, firstNames=Bob, lastName=null, sex=null, dateOfDeath=null, maidenName=null, dateOfBirth=null, address=null, birthPlace=null, deathPlace=null, occupation=null, retired=null)), links: [</events/a5383689-1192-4078-a4a6-a611b0a34c6e>;rel="self"] }, EntityModel { content: EventNotification(eventId=ec39aa80-2fa2-4d46-9211-c66fc94024d3, eventType=DEATH_NOTIFICATION, sourceId=ec39aa80-2fa2-4d46-9211-c66fc94024d3, dataIncluded=null, enrichmentFields=null, eventData=DeathNotificationDetails(registrationDate=null, firstNames=Bob, lastName=Smith, sex=null, dateOfDeath=null, maidenName=null, dateOfBirth=null, address=null, birthPlace=null, deathPlace=null, occupation=null, retired=null)), links: [</events/ec39aa80-2fa2-4d46-9211-c66fc94024d3>;rel="self"] }], fallbackType: null, metadata: Metadata { number: 0, total pages: 1, total elements: 2, size: 10 }, links: </events?filter[eventType]=DEATH_NOTIFICATION&page[number]=0&page[size]=10>;rel="self" }
+PagedModel { content: [EntityModel { content: EventNotification(eventId=a5383689-1192-4078-a4a6-a611b0a34c6e, eventType=DEATH_NOTIFICATION, sourceId=a5383689-1192-4078-a4a6-a611b0a34c6e, dataIncluded=null, enrichmentFields=null, eventData=DeathNotificationDetails(registrationDate=null, firstNames=Bob, lastName=null, sex=null, dateOfDeath=null, maidenName=null, dateOfBirth=null, address=null, birthPlace=null, deathPlace=null, occupation=null, retired=null)), links: [<https:/events/a5383689-1192-4078-a4a6-a611b0a34c6e>;rel="self"] }, EntityModel { content: EventNotification(eventId=ec39aa80-2fa2-4d46-9211-c66fc94024d3, eventType=DEATH_NOTIFICATION, sourceId=ec39aa80-2fa2-4d46-9211-c66fc94024d3, dataIncluded=null, enrichmentFields=null, eventData=DeathNotificationDetails(registrationDate=null, firstNames=Bob, lastName=Smith, sex=null, dateOfDeath=null, maidenName=null, dateOfBirth=null, address=null, birthPlace=null, deathPlace=null, occupation=null, retired=null)), links: [<https:/events/ec39aa80-2fa2-4d46-9211-c66fc94024d3>;rel="self"] }], fallbackType: null, metadata: Metadata { number: 0, total pages: 1, total elements: 2, size: 10 }, links: <https:/events?filter%5BeventType%5D=DEATH_NOTIFICATION&page%5Bnumber%5D=0&page%5Bsize%5D=10>;rel="self" }
```

Comment:
`https:/events/ec39aa80-2fa2-4d46-9211-c66fc94024d3` is silly here, but I think it's ok. I think this should be JSON as part of an integration test, which fixes this problem